.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+name: ci
+
+on:
+  pull_request:
+    branches: [main]
+  push:
+    branches: [main]
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: "read"
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

Comment:
should probably use v3 as v2 uses node 14, and v3 uses node 16

Reply:
Good catch!